### ISSUE TITLE
Plex Media Player stuck connecting

### ISSUE THREAD

**Issue by colonelkrud** [samcm/synclounge#76]
In Firefox Nightly (62.0a1 (2018-06-13) (64-bit)) on my mac (10.13.5), I am getting the following error when trying to connect to Plex Media Player (3.57.1) from v2.synclounge.tv:
Pulled from console:
`XML Parsing Error: no root element found
Location: http://10.0.0.2:32433/player/timeline/subscribe?port=8555&protocol=http&X-Plex-Device-Name=SyncLounge&commandID=1
Line Number 1, Column 1: subscribe:1:1
XML Parsing Error: no root element found
Location: http://10.0.0.179:32433/player/timeline/subscribe?port=8555&protocol=http&X-Plex-Device-Name=SyncLounge&commandID=3
Line Number 1, Column 1:`

**Comment by ThePokeX** [comment]
On Windows 10 using 64-Bit Google Chrome (67.0.3396.99) I cannot connect to the Windows desktop Plex app (3.57.1) when using v2.synclounge.tv
[image](https://user-images.githubusercontent.com/30876529/41947009-203ae0ba-7983-11e8-83fb-4cb38e665a76.png)
Issue is reproducible, so I can provide more information if needed.

**Comment by samcm** [comment]
Just to clarify, does connecting to PMP work when using v1 (app.synclounge.tv)? I've had a couple issues using PMP with v2 myself but I can't nail down why.

**Comment by ThePokeX** [comment]
It connects flawlessly.
![image](https://user-images.githubusercontent.com/30876529/42001901-77f902fe-7a33-11e8-873d-6733822e7625.png)

**Comment by colonelkrud** [comment]
V1 works for me also. Funnily enough v2 does work with Plex iOS and Apple TV for v2. Not sure if this is at all related to how v2 talks with Plex apps.

**Comment by samcm** [comment]
Thanks, looking in to this now

**Comment by colonelkrud** [comment]
Update: I was able to get SyncLounge to talk to PMP by repeatedly refreshing SyncLounge and restarting PMP. This only seems to work some of the time and I don’t have any logs that explain this process. None of the software components have updated. Important side note, I have a pretty powerful IDS and firewall and the SyncLounge server runs behind a NAT and reverse proxy. There have been no logs showing something being blocked and poke is an off site user. Connections to the SyncLounge server have been fine once established but the initial connection to the player has been hit or miss.

**Comment by samcm** [comment]
Thanks for the update. I think in a few cases PMP isn't updating its client details with Plex.tv - causing SyncLounge to connect to old details. That doesn't cover all cases however. Last week I couldn't connect even with up to date details but reinstalling PMP managed to get it going for a while. AppleTV has recently started facing the same issues so I'm wondering if something has been added that we're missing. 

Until I can find a fix I'd suggest either using OpenPHT or the inbuilt client (which can now direct stream most formats when enabled in SyncLounge Settings).

**Comment by samcm** [comment]
This should be fixed now, please let me know if you have any issues.

**Comment by TheThreemagi** [comment]
I am seeing this issue still, with the latest update there doesn't seem to use v1 or there update was also pushed to v1.  Either way app.synclounge.tv and v2.synclounge.tv do not work with plex media player at all for me or my friends.

**Comment by samcm** [comment]
For now I've rolled back app.synclounge.tv to v1, although my connectivity to PMP works on both v1 and v2. It has been a little inconsistent in general, although those issues were occurring across both versions.

@TheThreemagi Which view are you using with PMP? I seem to have better results with the TV mode.

**Comment by TheThreemagi** [comment]
I am using TV Mode, I just played around with switching just to see if web mode had anything different, no luck in either mode.

**Comment by samcm** [comment]
I've just committed a potential fix for this which should be live on v2.synclounge.tv within the hour. Please let me know if it works when you get a chance. I'm not 100% certain but here's hoping. 

Funnily enough my PMP has the exact same issue with v1 and v2. It will randomly start working after I leave it open for a few minutes.

**Comment by TheThreemagi** [comment]
Just checked now, not sure if the commit was pushed out but it still has the same behavior.  Not sure if it helps but I did figure out some oddities in the connected.  If I try to connect to v2 and it doesn't work then I got back to v1, v1 will not connect unless I restart PMP.  If on I try v1 first and connect and then connect to v2, v2 works perfectly till I restart PMP.   Hopefully that's helpful.
